.github/workflows/test.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+
+name: Testing
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  test:
+    name: Test with Ruby-${{ matrix.ruby }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # Due to https://github.com/actions/runner/issues/849, we have to use quotes for '3.0'
+        # TODO: Add jruby if something like allow_failures will be implemented on Actions.
+        ruby: [2.3, 2.4, 2.5, 2.6, 2.7, '3.0', head]
+    env:
+      GITHUB_WORKSPACE: ${{ github.workspace }}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby }}
+        bundler-cache: true
+    - name: Install dependencies
+      run: sudo apt-get install -y pandoc nodejs pkg-config libxml2-dev libxslt-dev
+    - name: Run tests
+      run: ./.actions.sh
+
+# TODO(zzak, rkh): Add slack notification.
```

Comment:
@zzak Probably we need to set up an integration with slack but I don't have a permission to generate a token on slack. Could you work on this?